packages/salesforcedx-vscode-lwc/src/index.ts

```diff
@@ -25,21 +31,28 @@ export async function activate(context: ExtensionContext) {
     return;
   }
 
+  const workspaceType = lwcLanguageServer.detectWorkspaceType(
+    workspace.workspaceFolders[0].uri.path
+  );
+
   // Check if ran from a LWC project
-  if (
-    !lwcLanguageServer.isLWC(
-      lwcLanguageServer.detectWorkspaceType(
-        workspace.workspaceFolders[0].uri.path
-      )
-    )
-  ) {
+  if (!lwcLanguageServer.isLWC(workspaceType)) {
     console.log('Not a LWC project, exiting extension');
     return;
   }
 
-  // The debug options for the server
-  const debugOptions = { execArgv: ['--nolazy', '--inspect=6009'] };
+  startLWCLanguageServer(serverModule, context);
 
+  if (workspaceType === lwcLanguageServer.WorkspaceType.SFDX) {
```

Comment:
Only activate for SFDX workspace since internal users have eslint installed at a different location.